Common/Time.cs

```diff
@@ -221,6 +222,17 @@ public static DateTime Min(DateTime one, DateTime two)
             return one < two ? one : two;
         }
 
+        /// <summary>
+        /// Multiplies the specified interval by the multiplier
+        /// </summary>
+        /// <param name="interval">The interval to be multiplied, such as TimeSpan.FromSeconds(1)</param>
+        /// <param name="multiplier">The number of times to multiply the interval</param>
+        /// <returns>The multiplied interval, such as 1s*5 = 5s</returns>
+        public static TimeSpan Multiply(this TimeSpan interval, double multiplier)
```

Comment:
Help convert indicator numerical periods into timespans.

Reply:
and just plain awesome :) kind of surprised the .net framework doesn't support this operation